queue-conversation-notification/nodejs/main.js

```diff
@@ -0,0 +1,89 @@
+const platformClient = require('purecloud-platform-client-v2');
+var WebSocketClient = require('websocket').client;
+
+// Genesys Cloud Credentials
+const GENESYS_CLOUD_CLIENT_ID = process.env.GENESYS_CLOUD_CLIENT_ID;
+const GENESYS_CLOUD_CLIENT_SECRET = process.env.GENESYS_CLOUD_CLIENT_SECRET;
+const GENESYS_CLOUD_ENVIRONMENT = 'mypurecloud.com';
```

Comment:
Don't hard code the value.  Read it from an environment variable

Reply:
Modified hradcoded environment/region values to use environment variables.